.github/workflows/notify-release.yml

```diff
@@ -45,22 +45,7 @@ jobs:
         if: steps.firefox.outputs.is_new
         uses: Ilshidur/action-discord@f1ed8844d9b33c17221fab0f36672cde39800eed
         env:
-          DISCORD_WEBHOOK: ${{ secrets.DISCORD_UX_WEBHOOK }}
-          DISCORD_USERNAME: Hiro Team
-          DISCORD_AVATAR: 'https://i.imgur.com/z9Iy6ug.png'
-          DISCORD_EMBEDS: |
-            [{
-              "title": "Leather for Firefox",
-              "url": "https://addons.mozilla.org/en-US/firefox/addon/hiro-wallet/"
-            }]
-        with:
-          args: ':rocket: A new version (${{ steps.firefox.outputs.new_version }}) of the Leather is available on the Firefox Web Store!'
-
-      - name: Firefox Discord notification - userx-notifs
-        if: steps.firefox.outputs.is_new
-        uses: Ilshidur/action-discord@f1ed8844d9b33c17221fab0f36672cde39800eed
-        env:
-          DISCORD_WEBHOOK: ${{ secrets.DISCORD_UX_NOTIFS_WEBHOOK }}
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_LEATHER_WEBHOOK }}
           DISCORD_USERNAME: Hiro Team
```

Comment:
Do we need to have a new user for `Leather` now too instead of `Hiro Team`?

Reply:
yesss